README.md

```diff
@@ -22,9 +30,29 @@ async function run () {
 run()
 ```
 
-First, the script checks whether `ILP_CREDENTIALS` is defined in the environment.
-`ILP_CREDENTIALS` must contain a JSON object with the options passed into the
-constructor of `ilp-plugin-btp` or the module name in `ILP_PLUGIN`.
+TypeScript:
+```typescript
+import createPlugin, { DataHandler } from '..'
+const plugin = createPlugin()
+const echo: DataHandler = (data: Buffer) => {
+  return Promise.resolve(data)
+}
+
+async function run () {
+  plugin.registerDataHandler(echo)
+  await plugin.connect()
+  await plugin.sendData(/* ... */)
+  process.exit(0)
+}
+
+run()
+```
+
+If no parameters are provided it will attempt to find the config in environment variables. If these are not found it will load a plugin that attempts to connect to a local [moneyd](../moneyd) instance on port 7768.
```

Comment:
I meant link to the moneyd repo so you can find instructions for running it (right now the link is broken)